infrastructure/aat.tfvars

```diff
@@ -3,7 +3,7 @@ external_host_name = "gateway-ccd.aat.platform.hmcts.net"
 idam_api_url = "https://idam-api.aat.platform.hmcts.net"
 cors_origin = "https://www-ccd.nonprod.platform.hmcts.net,http://localhost:3451,*"
 document_management_url = "http://dm-store-aat.service.core-compute-aat.internal"
-mv_annotations_api_url = "https://em-anno-aat.service.core-compute-aat.internal"
```

Comment:
@MSancaktutar just came to my mind that soon we will switch in Prod to AKS. 
Have you got some tickets on your backlog to configure media viewer and annotations for AKS? i.e. adding config to the charts

Reply:
EM team will let us know, if they change. And we'll definitely adjust our configs after them or us moving to AKS.